projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.js

```diff
@@ -0,0 +1,27 @@
+window.onload = function () {
+	const modal = document.getElementById( 'wpcom-subscribe-modal' );
```

Comment:
Let's bail out early if the element isn't found for any reason.

Reply:
Added this.